src/services/forecast.ts

```diff
@@ -28,16 +28,15 @@ export class Forecast {
   ) {}
 
   public async processForecastForBeaches(
-    beaches: Beach[]
+    beaches: Beach[],
+    query: 'asc' | 'desc' = 'desc'
```

Comment:
Maybe `orderBy` is better than `query` here?

Reply:
sure ! i will change it.